config/fcos/v1_5_exp/validate_test.go

```diff
@@ -187,3 +187,44 @@ func TestValidateBootDevice(t *testing.T) {
 		})
 	}
 }
+
+// TestValidateExtension tests extension validation
+func TestValidateExtension(t *testing.T) {
+	tests := []struct {
+		in      Extension
+		out     error
+		errPath path.ContextPath
+	}{
+		// No name field
+		{
+			Extension{},
+			common.ErrPackageNameRequired,
+			path.New("yaml", "extensions"),
+		},
+		// complete config
+		{
+			Extension{
+				Name: "strace",
+			},
+			nil,
+			path.New("yaml", "extensions"),
```

Comment:
To avoid being misleading, let's just do this for nil error cases:

```suggestion
			path.New("yaml"),
```